<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="input_trajectory_topic" default="/planning/trajectory/optimizer"/>
  <arg name="output_trajectory_topic" default="/planning/trajectory"/>
  <arg name="input_odometry_topic" default="/localization/kinematic_state"/>

  <arg name="time_to_space_converter_param_path" default="$(find-pkg-share autoware_time_to_space_trajectory_converter)/config/time_to_space_trajectory_converter.param.yaml"/>

  <node pkg="autoware_time_to_space_trajectory_converter" exec="time_to_space_trajectory_converter" name="time_to_space_trajectory_converter" output="screen">
    <param from="$(var time_to_space_converter_param_path)" allow_substs="true"/>
    <remap from="~/time_to_space_converter/input/trajectory" to="$(var input_trajectory_topic)"/>
    <remap from="~/time_to_space_converter/input/odometry" to="$(var input_odometry_topic)"/>
    <remap from="~/time_to_space_converter/output/trajectory" to="$(var output_trajectory_topic)"/>
  </node>
</launch>
